qucs/qucs-powercombining/qucspowercombiningtool.cpp

```diff
@@ -0,0 +1,1189 @@
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "qucspowercombiningtool.h"
+#include<./../qucs/qucs.h>
```

Comment:
You should use double quotes here. `<...>` symbols usually are intended for system headers. See https://github.com/andresmmera/qucs/pull/2 for fix